kolibri/plugins/learn/assets/src/routes/classesRoutes.js

```diff
@@ -1,39 +1,54 @@
 import store from 'kolibri.coreVue.vuex.store';
-import { ClassesPageNames } from '../constants';
+import router from 'kolibri.coreVue.router';
+import { ClassesPageNames, PageNames } from '../constants';
 import { showLessonPlaylist, showLessonResourceViewer } from '../modules/lessonPlaylist/handlers';
 import { showClassAssignmentsPage } from '../modules/classAssignments/handlers';
 import { showAllClassesPage } from '../modules/classes/handlers';
 import { showExam } from '../modules/examViewer/handlers';
 import { showExamReport } from '../modules/examReportViewer/handlers';
 
```

Comment:
I think we need to place a reverse version of this guard on the `TOPICS_ROOT` and related routes. That is, if `!canAccessUnassignedContent`, then redirect to the `/classes` URL